rotkehlchen/data/importer.py

```diff
@@ -469,3 +480,269 @@ def import_cryptocom_csv(self, filepath: Path) -> Tuple[bool, str]:
                 except KeyError as e:
                     return False, str(e)
         return True, ''
+
+    def _consume_blockfi_entry(self, csv_row: Dict[str, Any]) -> None:
+
+        if len(csv_row['Confirmed At']) != 0:
+            timestamp = deserialize_timestamp_from_date(
+                date=csv_row['Confirmed At'],
+                formatstr='%Y-%m-%d %H:%M:%S',
+                location='BlockFi',
+            )
+        else:
+            raise DeserializationError('Action not confirmed yet')
```

Comment:
What does that error mean? Imagine seeing that in the frontend:

`Error during BlockFi CSV import deserialization. Error was Action not confirmed yet. Ignoring entry`

What should the user understand from this?

Can that happen? Does not confrmed mean that it has not been "mined"? Why not just skip it completely and just not show anything to the user? It's not an error. It's just something that's not done yet.